src/Compilers/Core/Portable/Emit/EditAndContinue/EncMappedMethod.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.CodeAnalysis.Symbols;
+
+namespace Microsoft.CodeAnalysis.Emit;
+
+internal readonly struct EncMappedMethod(IMethodSymbolInternal previousMethod, Func<SyntaxNode, SyntaxNode?>? syntaxMap, Func<SyntaxNode, RuntimeRudeEdit?>? runtimeRudeEdit)
+{
+    public readonly IMethodSymbolInternal PreviousMethod = previousMethod;
+    public readonly Func<SyntaxNode, SyntaxNode?>? SyntaxMap = syntaxMap;
+    public Func<SyntaxNode, RuntimeRudeEdit?>? RuntimeRudeEdit { get; } = runtimeRudeEdit;
```

Comment:
>{ get; } [](http://example.com/codeflow?start=63&length=8)

Just curious: Why a mix of fields and properties, here and across the new `Enc*Info` and `Enc*Value` types?

Reply:
Fixed. Refactoring added it as a property.